Resources/Locale/en-US/gases/gases.ftl

```diff
@@ -4,6 +4,6 @@ gases-co2 = Carbon Dioxide
 gases-plasma = Plasma
 gases-tritium = Tritium
 gases-water-vapor = Water Vapor
-gases-ammonia = Ammonia
+gases-ammonia = Miasma
```

Comment:
you didnt rename the gas abbreviation, which is NH3 for ammonia.

I dont remember what it used to be but i imagine miasma's abbreviation was something like `Mias`. you can check the git history for that.